eng/Versions.props

```diff
@@ -245,11 +245,11 @@
     <GrpcAuthVersion>2.43.0</GrpcAuthVersion>
     <GrpcNetClientVersion>2.43.0</GrpcNetClientVersion>
     <GrpcToolsVersion>2.43.0</GrpcToolsVersion>
-    <DuendeIdentityServerAspNetIdentityVersion>5.2.0</DuendeIdentityServerAspNetIdentityVersion>
-    <DuendeIdentityServerEntityFrameworkVersion>5.2.0</DuendeIdentityServerEntityFrameworkVersion>
-    <DuendeIdentityServerVersion>5.2.0</DuendeIdentityServerVersion>
-    <DuendeIdentityServerStorageVersion>5.2.0</DuendeIdentityServerStorageVersion>
-    <DuendeIdentityServerEntityFrameworkStorageVersion>5.2.0</DuendeIdentityServerEntityFrameworkStorageVersion>
+    <DuendeIdentityServerAspNetIdentityVersion>6.0.4</DuendeIdentityServerAspNetIdentityVersion>
```

Comment:
I think this variable is not used anywhere nor I was able to find a package with that name

Reply:
The package should be: https://www.nuget.org/packages/Duende.IdentityServer.AspNetIdentity/6.0.4 do you see that?

Reply:
I can see the NuGet but this property is not used anywhere in the repo so maybe it's not needed?

Reply:
Ok I'll try a build

Reply:
Definitely need this package:

```
src/Identity/ApiAuthorization.IdentityServer/src/Microsoft.AspNetCore.ApiAuthorization.IdentityServer.csproj(0,0): error NU1102: (NETCORE_ENGINEERING_TELEMETRY=Restore) Unable to find package Duende.IdentityServer.AspNetIdentity with version (>= 6.0.4)
  - Found 20 version(s) in dotnet-public [ Nearest version: 5.2.1 ]
```

Reply:
Oh sorry, it didn't get added. It's there now!

Reply:
Thanks, looks like we also need Automapper 11 as a result of these updated packages too...

https://www.nuget.org/packages/AutoMapper/

Reply:
Done

Reply:
Another updated package: IdentityModel 6.0.0  https://www.nuget.org/packages/IdentityModel/